frontend/src/components/pages/admin/UsersPage.tsx

```diff
@@ -187,6 +187,7 @@ const UsersPage = (): React.ReactElement => {
     setTabIndex(index);
   };
 
+  type Role = "teacher" | "admin";
```

Comment:
can you reuse the `Role` type defined in `types/AuthTypes.ts` instead of redefining here?

Reply:
oh this was a merge conflict, I didn't actually touch this page. Do you want me to delete that line?

Reply:
yes please :)